ramls/instance.json

```diff
@@ -63,7 +69,23 @@
       "type": "array",
       "description": "List of series titles associated with the resource (e.g. Harry Potter)",
       "items": {
-        "type": "string"
+        "type": "object",
+        "properties": {
+          "value": {
+            "type": "string",
+            "description": "Series title value"
+          },
+          "authorityId": {
+            "type": "string",
+            "description": "UUID of authority record that controls an series title",
+            "$ref": "uuid.json",
+            "readonly": true
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "value"
+        ]
```

Comment:
Thanks @julianladisch for raising this. I agree that running this migration asynchronously potentially means that the intended representation and the stored representation are incompatible.

Reply:
@julianladisch, our approach is to make everything in one platform update process: update the module to the new version, run migration job, and then run reindex to update data in search engine.
If we will support both format and run asynchronously migration, it will cause some problems:
* some mechanism should exist to convert from one format to another on fly
* reindex process should be adjusted to index from both old and new fields
* possibly some problems with data-import (for example for records that were not yet migrated but data-import tries to update subjects field in new format)
* how to be with clients that will expect new format
* much more effort to support both fields

Please describe, how do you see this async process with both supported fields and what points to do it?

Reply:
Option 3:
1. Leave API changes as it is.
2. Made changes to GET endpoints of mod-inventory-storage  related to this breaking change to modify database data from a string array to an object array

Pros of the option:
1. No API changes in the next release.
2. No additional fields.
3. No additional changes to clients.
4. No additional changes to data-import mapping and search reindex processes.
5. Module could be used just after reindex is done. After reindexing, the MODINVSTOR-1012 migration job could be initialized that will not block using of the module.